manifest.md

```diff
@@ -122,6 +130,76 @@ Unlike the [image index](image-index.md), which contains information about a set
 }
 ```
 
+## Guidelines for Artifact Usage
+
+Content other than OCI container images MAY be packaged using the image manifest.
+When this is done, the `config.mediaType` value MUST be set to a value specific to the artifact type.
+For portability, if the artifact does not have a content for the config blob, an empty json object, `{}`, SHOULD be pushed for the blob content.
+If the artifact does not need layers, a single layer SHOULD be included with a non-zero size.
+The suggested content for an unused layer is the empty json object, `{}`.
```

Comment:
Can't we just use "" (0 byte) ?

Reply:
I wanted to, but #1025 found compatibility issues in the "zero layer" manifest.